File: pgm/ABC_Dart/ABC_427_A.dart
import 'dart:io';
// import 'dart:math';

void main() {
  final String S = stdin.readLineSync()!;
  const int one = 1;
  const int two = 2;
  const int zero = 0;
  final int idx = (S.length + one) ~/ two - one;
  print(S.substring(zero, idx) + S.substring(idx + one));
  exit(0);
}

File: pgm/ABC_Java/ABC_427_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final String S = reader.readLine();
        reader.close();
        final int one = 1;
        final int two = 2;
        final int zero = 0;
        System.out.println (S.substring (zero, S.length() / two)
                                .concat (S.substring (S.length() / two + one)));
        System.exit (0);
    }
}